test-data/unit/check-final.test

```diff
@@ -487,16 +484,16 @@ class C:
 [out]
 
 [case testFinalReassignInstanceVarClassVsInit]
-# flags: --no-new-semantic-analyzer
+# flags: --new-semantic-analyzer
 # Methods are processed after top-level in new analyzer.
 from typing import Final
 
 class C:
     y: Final = 1
     def __init__(self) -> None:
-        self.x: Final = 1
+        self.x: Final = 1  # E: Cannot redefine an existing name as final
```

Comment:
The error here is a bit confusing. Maybe move the comment at the top of the test case, right above the error, so that the former is more prominent?